tests/storage/test_storage.py

```diff
@@ -1003,6 +1003,12 @@ def test_fetch_url_with_version(self):
         fetch_url = store.fetch_url("some/key", version_id="some-version")
         assert fetch_url == "/media-url/bucket-name/some/some-version/key"
 
+    @override_settings(MEDIA_ROOT="/media-root/", MEDIA_URL="/media-url/")
+    def test_fetch_url_with_key_path_containing_root_and_bucket(self):
+        store = storage.LocalFileStore("bucket-name")
+        fetch_url = store.fetch_url("/media-root/bucket-name/some/key")
```

Comment:
would you not use the setting here rather than hard code the media root into the arg?

Reply:
also I don't think we are passing it the bucket in the url key path are we?